src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView.java

```diff
@@ -766,18 +766,7 @@ public Item doCreateItem(final StaplerRequest req, final StaplerResponse rsp) th
     @Override
     public boolean hasPermission(final Permission p) {
         try {
-            boolean display = true;
-            //tester la liste vide seulement en lecture
-            if (READ.name.equals(p.name)) {
-                if (isEmpty()) {
-                    display = false;
-                }
-            } else {
-                //Pas en lecture => permission standard
-                display = super.hasPermission(p);
-            }
-
-            return display;
+            return isEmpty() ? this.getACL().hasPermission(View.CONFIGURE) : super.hasPermission(p);

```

Comment:
I believe you just want to delegate to the ACL? It looks like it repeats this logic - first check permission with the base, if not found but items are empty and we want READ, consult parent's CONFIGURE permission.

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/security/AuthorizationStrategy.java#L102